### PR TITLE
Added Yii2 recipes

### PR DESCRIPTION
**Pull request by arogachev**

### CODE REVIEW

File: recipe/yii2-app-basic.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+/**
+ * Main task
+ */
+task('deploy', [
+    'deploy:prepare',
+    'deploy:release',
+    'deploy:update_code',
+    'deploy:shared',
+    'deploy:vendors',
+    'deploy:run_migrations',
```
Comment (elfet) [inline recipe/yii2-app-basic.php]: I think it's better to do not include this in default `deploy` task.

Comment (arogachev) [inline recipe/yii2-app-basic.php]: Why? It's required (not optional) operations for correct deploy.

**Comment by elfet** [comment]
Ok, i'll merge this :)

**Comment by elfet** [comment]
@oanhnn may be it's time for new release? :eyes:

**Comment by arogachev** [comment]
:+1:

**Pull request merged**